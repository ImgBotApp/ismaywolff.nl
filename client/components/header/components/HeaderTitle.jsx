import styled from 'styled-components'

const HeaderTitle = styled.h1`
  white-space: nowrap;
  order: 1;
  width: 100%;
  text-align: center;
  margin-top: 0;
  margin-bottom: 0;

  @media (min-width: 20em) {
    margin-left: 1rem;
    margin-right: 1rem;
    width: auto;
    order: unset;
  }
`

export default HeaderTitle
