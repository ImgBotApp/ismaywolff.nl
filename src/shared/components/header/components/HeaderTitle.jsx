import styled from 'styled-components'

const HeaderTitle = styled.h1`
  font-family: Bitter, serif;
  font-size: 1.5rem;
  margin-bottom: 0.5rem;
  margin-top: 0;
  text-align: center;
  white-space: nowrap;

  @media (min-width: 30em) {
    margin-bottom: 0;
    text-align: left;
  }
`

export default HeaderTitle
