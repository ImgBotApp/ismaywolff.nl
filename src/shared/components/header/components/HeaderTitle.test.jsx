import React from 'react'
import { shallow } from 'enzyme'
import HeaderTitle from './HeaderTitle'

describe('<HeaderTitle />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<HeaderTitle />)
    expect(wrapper).toMatchSnapshot()
  })
})
