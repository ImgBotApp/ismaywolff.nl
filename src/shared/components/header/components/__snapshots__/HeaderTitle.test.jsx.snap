// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderTitle /> renders correctly 1`] = `
.c0 {
  font-size: 1.5rem;
  margin-bottom: 0.5rem;
  margin-top: 0;
  text-align: center;
  white-space: nowrap;
}

@media (min-width: 30em) {
  .c0 {
    margin-bottom: 0;
    text-align: left;
  }
}

<h1
  className="c0"
/>
`;
